packages/discovery-provider/plugins/pedalboard/apps/mri/src/index.ts

```diff
@@ -0,0 +1,43 @@
+import knex from "knex"
+import { S3Client } from "@aws-sdk/client-s3"
+import { logger } from "./logger"
+import { CronJob } from "cron"
+import { webServer } from "./server"
+import { createS3Instances } from "./s3"
+import { clm } from "./clm"
+import { readConfig } from "./config"
+
+const main = async () => {
+    logger.info({}, "good morning")
+
+    const config = readConfig()
+
+    const db = knex({
+        client: 'pg',
+        connection: config.dbUrl
+    })
+
+    const s3s = await createS3Instances()
+
+    const job = CronJob.from({
```

Comment:
doesn't pedalboard support a cron feature?

Reply:
it does but it's not backed by this library, i'd rather have pedalboard use this as a backend as it's got more features

Reply:
sgtm!